### PR TITLE
IPC-322 HealthAPILibrary and HealthSDK XCFrameworks

### PR DESCRIPTION
**Pull request by antosj**

### CODE REVIEW

File: HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/GiniHealthUtils.swift [hunk 1]
```diff
@@ -11,7 +11,7 @@ import UIKit
  
  */
 public func giniHealthBundle() -> Bundle {
-    Bundle.module
+    Bundle.resource
```
Comment (razvancapra) [inline HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/GiniHealthUtils.swift:14]: Clarification: Why we need this change? 
We have down here the `giniHealthBundleResource()` function that's providing the resources.

**Review by razvancapra — CHANGES_REQUESTED** [review]
Look good. 
Just a small clarification.

File: HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/GiniHealthUtils.swift [hunk 2]
```diff
@@ -11,7 +11,7 @@ import UIKit
  
  */
 public func giniHealthBundle() -> Bundle {
-    Bundle.module
+    giniHealthBundleResource()
```
Comment (caprapaditu) [inline HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/GiniHealthUtils.swift:14]: Regarding: `giniHealthBundle()`
I would rather leave that as it is and I would use `giniHealthBundleResource()` in the `defaultImage()` function. I see there is the only place where is used.

**Review by caprapaditu — CHANGES_REQUESTED** [review]

File: HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/GiniHealthUtils.swift (hunk 2 above)
Comment (razvancapra) [inline HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/GiniHealthUtils.swift:14]: Regarding: `giniHealthBundle()`
I would rather leave that as it is and I would use `giniHealthBundleResource()` in the `defaultImage()` function. I see there is the only place where is used.

**Review by razvancapra — CHANGES_REQUESTED** [review]

**Review by razvancapra — APPROVED** [review]

**Pull request merged**